extensions/amp-viewer-integration/0.1/touch-handler.js

```diff
@@ -138,7 +138,9 @@ export class TouchHandler {
    * @private
    */
   forwardEvent_(e) {
-    if (e && e.shouldViewerCancelPropagation) {
+    // Check if an AMP component is signaling that we should
+    // stop propagation of the event from bubbling up to the viewer
+    if (e?.shouldViewerCancelPropagation) {
```

Comment:
To confirm, if passively modifying an event will always provide a signal to cancel propagation as necessary to prevent the issue.

Reply:
Adding the passive option to the event handler does not have anything to do stopping propagation. But also, to be fair, the handler does not do a lot of work, there's not a significant gain here anyways.

Reply:
Makes sense: For others reading here's the direct quote from the DOM Standard.

> When set to true, options’s passive indicates that the callback will not cancel the event by invoking preventDefault(). This is used to enable performance optimizations described in § 2.8 Observing event listeners.